### ISSUE TITLE
History Filters

### ISSUE THREAD

**Issue by ryanproud** [tegon/traktflix#72]
Some filters for the history would be nice.  ie. (not synced, missing trakt match, etc.)

**Comment by Sparticuz** [comment]
This would be nice, even just a CSS Display: none, if netflix watched and trakt watched are the same

**Comment by tegon** [comment]
I agree, but unfortunately, I'm struggling to find the time to work on this lately, so I'm not sure when I'll be able to do it.

**Comment by gsrafael01** [comment]
Since v2.0.0, traktflix informs you if the item is already synced or not found on Trakt, but doesn't hide them. I can definitely add an option to do that.

**Comment by R-K-H** [comment]
It might be useful as a default to only populate the list with non-synced as you could prevent making any requests out to get information for those and speed up performance. But it would depend on what people want and what they are expecting when clicking on history. 

Maybe there are two options for the extension, one is sync and one is history. That way when clicking history you are taken to history not history sync which might be a bit misleading? And then sync would only have missing items or items which were unable to locate a match etc.

**Comment by alexanderadam** [comment]
Also a hide list / block list would be perfect. Since I don't want to sync some series.
